### PR TITLE
Update 1.7.0

### PR DESCRIPTION
**Pull request by jvies**
Hi,

With this PR, we can propose the latest 1.7.0 for testing.

I've also replaced ghcr.io username by GITHUB_REPOSITORY_OWNER to reduce diffs between forks and your repo.

best regards,

### CODE REVIEW

**Comment by mriedmann** [comment]
I don't exactly know why the builds are failing but I suspect some kind of security feature. I am quite new to Github-Jobs, do you know what's going on there?

File: .github/workflows/dockerimage.yml [hunk 1]
```diff
@@ -71,12 +71,12 @@ jobs:
...
         echo "HUMHUB_VERSION=$HUMHUB_VERSION" >> $GITHUB_ENV
     - name: Login to registry
-      run: echo "${{ secrets.PAT }}" | docker login ghcr.io --username mriedmann --password-stdin
+      run: echo "${{ secrets.PAT }}" | docker login ghcr.io --username ${GITHUB_REPOSITORY_OWNER} --password-stdin
     - name: Pull images 
       run: |
-        docker pull ghcr.io/mriedmann/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-allinone
+        docker pull ghcr.io/${GITHUB_REPOSITORY_OWNER}/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-allinone
     - name: replace image
-      run: sed -i "s/docker.io\/mriedmann\/humhub:latest/ghcr.io\/mriedmann\/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-allinone/g" docker-compose.yml
```
Comment (mriedmann) [inline .github/workflows/dockerimage.yml:79]: please be careful here: As long as the docker-compose file is not changed the first part should stay static. From my point of view, this would break with the current compose file.

File: .github/workflows/dockerimage.yml [hunk 2]
```diff
@@ -101,15 +101,15 @@ jobs:
...
-        docker pull ghcr.io/mriedmann/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-phponly
-        docker pull ghcr.io/mriedmann/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-nginx
+        docker pull ghcr.io/${GITHUB_REPOSITORY_OWNER}/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-phponly
+        docker pull ghcr.io/${GITHUB_REPOSITORY_OWNER}/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-nginx
     - name: replace image
       run: |
-        sed -i "s/docker.io\/mriedmann\/humhub:stable-nginx/ghcr.io\/mriedmann\/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-nginx/g" docker-compose.prod.yml
-        sed -i "s/docker.io\/mriedmann\/humhub:stable-phponly/ghcr.io\/mriedmann\/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-phponly/g" docker-compose.prod.yml
+        sed -i "s/docker.io\/${GITHUB_REPOSITORY_OWNER}\/humhub:stable-nginx/ghcr.io\/${GITHUB_REPOSITORY_OWNER}\/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-nginx/g" docker-compose.prod.yml
+        sed -i "s/docker.io\/${GITHUB_REPOSITORY_OWNER}\/humhub:stable-phponly/ghcr.io\/${GITHUB_REPOSITORY_OWNER}\/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-phponly/g" docker-compose.prod.yml
```
Comment (mriedmann) [inline .github/workflows/dockerimage.yml:112]: Same here: `sed -i "s/docker.io\/${GITHUB_REPOSITORY_OWNER}\/humhub:stable-phponly/ghcr.io\/${GITHUB_REPOSITORY_OWNER}\/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-phponly/g"` => `sed -i "s/docker.io\/mriedmann\/humhub:stable-phponly/ghcr.io\/${GITHUB_REPOSITORY_OWNER}\/humhub:${GITHUB_REF_NAME_SLUG}-${HUMHUB_VERSION}-phponly/g"`

File: .github/workflows/dockerimage.yml (hunk 1 above)
Comment (jvies) [reply .github/workflows/dockerimage.yml:79]: sorry, wrong replace all.

File: .github/workflows/dockerimage.yml (hunk 2 above)
Comment (jvies) [reply .github/workflows/dockerimage.yml]: sorry, same wrong replace all.

**Comment by jvies** [comment]
I'm new to Github-Jobs too. This MR is my very first on a project based with Github-Jobs.

It seems it cannot get your secrets.PAT variable. Can you try to restart the jobs with your account ?

**Comment by mriedmann** [comment]
**TL;DR**: We need to add a new PR pipeline that is not using any secrets.

thx for the fix, I am answering my own question here: Access to secrets is bound to the "actor" which is (in case of this PR) @jvies how does not has access to the secret. This seems to be a security feature (that I like). The problem is that now the pipeline is a bit useless. We should fix this asap to make tasting 3rd party PRs testable again.

My idea is to split the pipeline into a PR build (not using the github registry) and leaving the current pipeline active on master. this would also provide us with a better and less wasteful tag-log on the github registry.

My time is quite limited for now. I would really appreciate any help with that!

**Comment by mriedmann** [comment]
@jvies thank you very much for your work.

I merged and released the 1.7.0/testing and 1.6.4/stable to docker hub with the new pipeline. Hopefully, this now works as expected. If you find any way to avoid the code duplication without major overhead, please let me know. For now, we will have to live with 2 separate pipelines for PRs and master-builds.
